lambda/proxy.test.ts

```diff
@@ -117,6 +117,26 @@ describe('proxy', () => {
     expect(axios.post).toHaveBeenCalled();
   });
 
+  it('should forward a request when header is Content-Type', async () => {
+    const destinationUrl = 'https://approved.host/github-webhook/';
+    const endpointId = encodeURIComponent(destinationUrl);
+    const event: APIGatewayProxyWithLambdaAuthorizerEvent<any> = {
+      ...baseEvent,
+      headers: {
+        ...baseEvent.headers,
+        'content-type': undefined,
```

Comment:
We should probably remove the `content-type` key/value from the test since it wouldn't appear in a real headers object

Reply:
I originally was doing a full `delete headers['content-type'];` but copilot suggested this as a way of doing the same thing and google searches seemed to confirm this does the same thing. Accurate?

Reply:
> in TypeScript, setting a key to undefined using object spread syntax does effectively remove the key from the resulting object.